### DISCUSSION TITLE
Display of Waypoints

### DISCUSSION THREAD

**Discussion by AndreasOSMAND** [osmandapp/OsmAnd#19382] [Q&A]
I have saved numerous waypoints from a gpx file in my favorites. I have marked them in different colors and marked them with different symbols.
In the display, some waypoints are now only shown as small dots, others with the symbol I have selected. If several waypoints have the same gps coordinates, only one is displayed.

I have not yet found a system for when which waypoint is displayed and how. 

Is it possible to set somewhere that the display always follows the one I have selected and, if there are several waypoints with the same GPS data, that a certain category is always displayed first?

I use Osmand 4.6.14 from F-Droid

**Comment by gbrown2036** [comment]
I believe I'm having the same problem.  Below is the XML structure of two of my waypoints in a GPX file.  They USED to display a different icon/color according what is in the <sym> tag (e.g. "Pin, Green" vs "Triangle, Blue") but now every waypoint looks the same.  Is this the same issue that you're having?


`<wpt lat="84.000000" lon="-84.000000"><ele>-1712.671508799575718</ele><time>2023-06-15T10:26:26Z</time><name>~GREATSTRUCTURE</name><cmt/>**<sym>Pin, Green</sym>**<type>user</type></wpt>`

`<wpt lat="27.000000" lon="-83.000000"><ele>-51.304954499457456</ele><time>2023-06-15T10:26:26Z</time><name>X029</name><cmt>BS JUNE 2018 GOOD SH  BS JUNE 2018 GOOD SH-BS JUNE 2018 GOOD SH</cmt>**<sym>Triangle, Blue</sym>**<type>user</type></wpt>`

**Reply by DmitryAlexei** [reply]
Probably related https://github.com/osmandapp/OsmAnd/discussions/19570

**Comment by AndreasOSMAND** [comment]
Hi gbrown2036 and DmitryAlexei,

thank you for your answers. They don't solve my problem but may be we are on the right way.

**Reply by gbrown2036** [reply]
This may help,
[List of standard values for `<osmand:icon>` `<osmand:background>` tags?](https://github.com/osmandapp/OsmAnd/discussions/19833#discussioncomment-9397820)
